docs/source/markdown/options/cap-add.md

```diff
@@ -5,3 +5,15 @@
 #### **--cap-add**=*capability*
 
 Add Linux capabilities.
+
+Granting additional capabilities increases the privileges of the
+processes running inside the container and potentially allow it to
+break out of confinement.  Capabilities like `CAP_SYS_PTRACE`,
+`CAP_MKNOD` and `CAP_SYS_MODULE` are particularly dangerous when they
```

Comment:
CAP_SYS_ADMIN should be added to this list.

Reply:
added